src/index.js

```diff
@@ -0,0 +1,190 @@
+const defaultCity = 'Seattle';
+const state = { temp: '', currentCity: defaultCity };
+
+const currentTemp = document.getElementById('current-temp');
+const upButton = document.getElementById('up-temp');
+const downButton = document.getElementById('down-temp');
+
+const currentLandscape = document.getElementById('landscape-weather-visual');
+const currentSky = document.getElementById('sky-weather-visual');
+const skySelector = document.getElementById('sky-select');
+
+const currentCityDisplay = document.getElementById('city');
+const cityInput = document.getElementById('city-input');
+const resetCityButton = document.getElementById('city-reset');
+
+const realTimeButton = document.getElementById('real-time-button');
+
+const dateTime = document.getElementById('date-time');
+const currentDate = new Date().toUTCString();
+dateTime.textContent = currentDate.slice(0, 3) + ' ' + currentDate.slice(5, 16);
+
+const select = document.querySelector('select');
+
+cityInput.placeholder = defaultCity;
+currentCityDisplay.textContent = state.currentCity;
+
+const setUpTemp = () => {
+  state.temp += 1;
+  displayTemp();
+  changeColor();
+  changeLandscape();
+  autoChangeSky();
```

Comment:
I see this block of code repeated on L37-40 and L160-162 as well. Maybe a good place to DRY with a helper function?